### PR TITLE
feat: move mnemonics out of disk

### PR DESCRIPTION
**Pull request by LuqiPan**
Closes: #139 

With the [workaround](https://github.com/Agoric/agoric-3-proposals/issues/136#issuecomment-2034174218) suggested by @gacogo, we no longer need to persist the mnemonics on disk. This PR removes the mnemonics on disk.

### CODE REVIEW

File: packages/synthetic-chain/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/synthetic-chain",
-  "version": "0.0.7",
+  "version": "0.0.8",
```
Comment (turadg) [inline packages/synthetic-chain/package.json:3]: we use Lerna to publish, which bumps the version. please leave out this commit. after the PR is green  I can cut and publish a release, pushing its commits

File: packages/synthetic-chain/public/upgrade-test-scripts/run_prepare_zero.sh [hunk 2]
```diff
@@ -12,8 +12,17 @@ export CHAINID=agoriclocal
 agd init localnet --chain-id "$CHAINID"
 
 allaccounts=("gov1" "gov2" "gov3" "user1" "validator")
-for i in "${allaccounts[@]}"; do
-  cat "/usr/src/upgrade-test-scripts/keys_for_test_only/$i.key" | agd keys add $i --recover --keyring-backend=test 2>&1
+# WARNING: these mnemonics are purely for testing purposes, do not implement
+# features that depend on them in any way.
+allkeys=(
```
Comment (turadg) [inline packages/synthetic-chain/public/upgrade-test-scripts/run_prepare_zero.sh:17]: may want to point out that the names come from matching indices in `allaccounts`

File: packages/synthetic-chain/src/cli/dockerfileGen.ts [hunk 3]
```diff
@@ -28,8 +28,6 @@ ENV UPGRADE_TO=${to}
 # put env functions into shell environment
 RUN echo '. /usr/src/upgrade-test-scripts/env_setup.sh' >> ~/.bashrc
 
-# provide files with keys for testing
-COPY --link ./upgrade-test-scripts/keys_for_test_only /usr/src/upgrade-test-scripts/keys_for_test_only
```
Comment (turadg) [inline packages/synthetic-chain/src/cli/dockerfileGen.ts:32]: nice cleanup

File: packages/synthetic-chain/package.json (hunk 1 above)
Comment (LuqiPan) [reply packages/synthetic-chain/package.json:3]: Awesome, if you could do a release on current `main` that'd be great. I was hoping to bring in the changes in https://github.com/Agoric/agoric-3-proposals/blob/main/packages/synthetic-chain/src/lib/constants.js and remove the reliance on https://github.com/Agoric/agoric-3-proposals/blob/main/proposals/16%3Aupgrade-8/agoric-synthetic-chain-0.0.7-alpha.tgz

Comment (turadg) [reply packages/synthetic-chain/package.json:3]: sorry, I forgot this repo works differently. we do manually bump and then simply `npm publish`. so if you want this to be 0.0.8 this is the right approach. it just won't be in NPM until someone with publish perms runs `npm publish`.

File: packages/synthetic-chain/public/upgrade-test-scripts/run_prepare_zero.sh (hunk 2 above)
Comment (LuqiPan) [reply packages/synthetic-chain/public/upgrade-test-scripts/run_prepare_zero.sh:18]: Good call, added a comment above

File: packages/synthetic-chain/package.json (hunk 1 above)
Comment (turadg) [reply packages/synthetic-chain/package.json:3]: published 604071875c0c398cdc4e4b59fbfc6a80a5e08998 as 0.0.8

**Review by turadg — APPROVED** [review]

**Pull request merged**